Dockerfile

```diff
@@ -43,7 +43,7 @@ RUN apt-get update -y && \
 RUN update-ca-certificates
 
 # Copy the compiled binaries over.
-COPY --from=builder /go/bin/sentinel /go/bin/arkeod /go/bin/directory-indexer /go/bin/directory-api /go/bin/tern /usr/bin/
+COPY --from=builder /go/bin/sentinel /go/bin/arkeod /go/bin/indexer /go/bin/api /go/bin/tern /usr/bin/
```

Comment:
feels weird to call them just "indexer" and "api". Thats kinda a non-sensical daemon name. Can we just merge the two daemons into one and call it `directory`?

Reply:
We could call the binary to whatever name, I change it is because the folder structure
Previous
```
-cmd
    |-directory
           |- directory-index
           |- directory-api
```
now
```
-cmd
    |-directory
           |- index
           |- api
```